hazelcast-jet-core/src/main/java/com/hazelcast/jet/impl/util/AsyncMapWriter.java

```diff
@@ -57,8 +59,6 @@
  */
 public class AsyncMapWriter {
```

Comment:
is this class still used anywhere?

Reply:
No, it's not. I kept it because we can use it for local imap sink.